### PR TITLE
Replaced shebang to point to bash instead of sh

### PR DESCRIPTION
**Pull request by pranaypratyush**
https://github.com/WpHeadless/wpheadless/issues/32

### CODE REVIEW

**Comment by tsertkov** [comment]
This will break cron functionality as bash is not available inside docker container running cron daemon.

File: compose.d/tasks.yml [hunk 1]
```diff
@@ -66,6 +66,6 @@ services:
           acf-to-rest-api=3.1.0
           wordpress-importer=0.6.4
           disable-comments=1.10.2
-          title-and-nofollow-for-links=1.09
-          wp-security-audit-log=4.0.0
-          activity-log-mainwp=1.4.1
+          title-and-nofollow-for-links=1.10
+          wp-security-audit-log=4.1.2
+          # activity-log-mainwp=1.4.1
```
Comment (tsertkov) [inline compose.d/tasks.yml:71]: Does not seem to be related to current PR.

Comment (pranaypratyush) [reply compose.d/tasks.yml:71]: OK. I understand. Better to remove pipefail entirely?

**Comment by pranaypratyush** [comment]
> This will break cron functionality as bash is not available inside docker container running cron daemon.

OK. I understand. Better to remove pipefail entirely?

File: compose.d/tasks.yml (hunk 1 above)
Comment (pranaypratyush) [reply compose.d/tasks.yml:71]: It is not. Let was just experimenting, would remove.

**Comment by tsertkov** [comment]
Closed in favour of af70105
